packages/node/README.md

```diff
@@ -0,0 +1,13 @@
+## Protocols
+
+The Node contains a TypeScript implementation of the [Counterfactual protocol](https://github.com/counterfactual/specs/blob/master/v0/protocols.md). It is responsible for executing the Counterfactual protocols [specified here](https://specs.counterfactual.com) and producing correctly constructed signed commitments that correspond to state transitions of the users' state channels.
+
+The specific design philosophy it adopts is the middleware pattern. That is, all of these protocols are naturally broken down into steps, for each of which there is a middleware responsible for executing that step.
```

Comment:
Not true for the Node in general.

Reply:
Yeah so this is all under the `Protocols` section of the README. It doesn't pertain to the Node in all its entirety.